### ISSUE TITLE
3.5.4 automatic ratio problem

### ISSUE THREAD

**Issue by luzrain** [sampotts/plyr#1437]
In new version 3.5.4, if not set ratio property, ratio set in 16:9 regardless of video resolution.

**Comment by sampotts** [comment]
Can you provide an example?

**Comment by luzrain** [comment]
>Can you provide an example?

Of course. The first image shows how player looks in 3.5.3 version. The second is version 3.5.4.
The purple block is equal to video size, the video block inside is stretched  to parent purple size.

![1](https://user-images.githubusercontent.com/25800964/57201407-4637f180-6fa1-11e9-9a60-076614c7a328.jpg)
![2](https://user-images.githubusercontent.com/25800964/57201408-4637f180-6fa1-11e9-8f5d-fbadd11f10a9.jpg)
